Azure/azure-cli
# [sql] `az sql midb move/copy`: Add new commands for Managed Database Move/Copy feature

**Related command**
These are new commands

**Description**

In this PR I introduce new commands to support SQL Managed Database Move/Copy feature. Managed databases can be moved/copied across managed instances (and resource groups) that whose vNets fulfill certain networking requirements.

The commands that are introduced are:

`az sql midb move start` - Start a managed database moveoperation from source instance to target instance 
`az sql midb move complete` - Complete a move operation
`az sql midb move cancel` - Cancel a move operation
`az sql midb move list` - List move operations from given instance and resource group (can also additionally be filtered by target rg and instance, and filtered to only include latest operation per managed database)
`az sql midb copy start` - Start a managed database copy operation from source instance to target instance
`az sql midb copy complete` - Complete a copy operation
`az sql midb copy cancel` - Cancel a copy operation
`az sql midb copy list` - List copy operations (same as list move operations)

**Testing Guide**

You can test operations by copying/moving managed database across managed instances in the same subnet (peering will be established and move/copy will work).

Some examples of commands:
`az sql midb move start --ids {id_of_database_to_move} --target-mi {name_of_target_mi}`
`az sql midb move complete --ids {id_of_database_to_move} --target-mi {name_of_target_mi}`
`az sql midb move cancel--ids {id_of_database_to_move} --target-mi {name_of_target_mi}`

Copy command are equivalent, instead of `move` use `copy`.


---

This checklist is used to make sure that common guidelines for a pull request are followed.
